tests/scanner/scanners/data/ke_jmespath_scanner_test_data.yaml

```diff
@@ -0,0 +1,115 @@
+rules:
```

Comment:
ditto, can you please leave `jmespath` out of the name and call this file to be `ke_scanner_test_data.yaml`?